src/models/multiBarChart.js

```diff
@@ -429,8 +674,25 @@ nv.models.multiBarChart = function() {
             rightAlignYAxis = _;
             yAxis.orient( rightAlignYAxis ? 'right' : 'left');
         }},
+        xTickFormat: {get: function(){return xAxis.xTickFormat();}, set: function(_){
```

Comment:
As mentioned in last comment, found similar implementation in lineWithFocusChart. This can be safely removed as well, if it is not required to have these functions externalised.